static/app/views/ddm/context.tsx

```diff
@@ -73,6 +73,17 @@ const emptyWidget: MetricWidgetQueryParams = {
   title: undefined,
 };
 
+function useCurrentProjects(): Project[] {
+  const {selection} = usePageFilters();
+  const {projects: projectIds} = selection;
+  const {projects} = useProjects();
+
+  return useMemo(() => {
+    const projectIdLookup = new Set(projectIds.map(id => id.toString()));
+    return projects.filter(project => projectIdLookup.has(project.id));
+  }, [projects, projectIds]);
+}
```

Comment:
I have not found an existing util for this, but it feels like there should be one 🤔 
Did I miss something @matejminar ?

Reply:
`useProjects` accepts an array of slugs to only fetch/return those - but in this case, we'd need to pass an array of IDs. I am not aware of a better way, unfortunately, our project slug vs project id string vs project id number situation leaves a lot to be desired 😔